src/ray/gcs/gcs_server/gcs_actor_manager.cc

```diff
@@ -623,7 +638,13 @@ void GcsActorManager::OnNodeDead(const ClientID &node_id) {
 
 void GcsActorManager::ReconstructActor(const ActorID &actor_id, bool need_reschedule) {
   auto &actor = registered_actors_[actor_id];
-  RAY_CHECK(actor != nullptr);
+  if (actor == nullptr) {
+    // NOTE(sang): It happens under this condition:
```

Comment:
It'd be nice to just write it out. Maybe something like, the actor was destroyed and deregistered before the node granted the actor creation request.

Reply:
I don't have a strong preference!